src/compiler/diagnosticMessages.json

```diff
@@ -439,10 +439,6 @@
         "category": "Error",
         "code": 1147
     },
-    "Cannot compile modules unless the '--module' flag is provided. Consider setting the 'module' compiler option in a 'tsconfig.json' file.": {

```

Comment:
Well that's a pain :-/  ModuleKind.None is 0, so I'll need to change all the existing checks that just looks for a falsy value as meaning "not set". Will look into it...